File: I-ŞansOyunu/SansOyunu.java
public class SansOyunu {
 public static void main(String arg[]) {
  boolean beyaz_top[] = new boolean[50];
  int beyaz, kirmizi, i, b;
  System.out.print("Beyaz Toplar: ");
  for (i = 0; i < 5; i++) {
   b = 0;
   while (b == 0) {
    beyaz = (int)(Math.random() * 48 + 1);
    if (beyaz_top[beyaz] == false) {
     System.out.print(beyaz + "\t");
     b = 1;
     beyaz_top[beyaz] = true;
    }
   }
  }
  kirmizi = (int)(Math.random() * 42 + 30);
  System.out.print("\nKirmizi Top: " + kirmizi);
 }
}

File: I-ŞansOyunu/SansOyunu.py
import random
beyaz_top = [False] * 50
print("Beyaz Toplar: ")
for i in range(0, 5):
    b = 0
    while b == 0: 
        beyaz = random.randint(1, 50)
        if beyaz_top[beyaz] == False:
            print(beyaz)
            b = 1
            beyaz_top[beyaz] = True;
    
kirmizi = random.randint(30, 73)
print("Kirmizi Top: ", kirmizi)